src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/GenotypeAlleleCounts.java

```diff
@@ -634,4 +648,25 @@ GenotypeAlleleCounts copy() {
                         }).collect(Collectors.toList());
     }
 
+
+
+    @FunctionalInterface
+    public interface TwoIntConsumer {
+        void accept(final int alleleIndex, final int alleleCount);
+    }
+
+    public void forEachAlleleIndexAndCount(final TwoIntConsumer action) {
+        new IndexRange(0, distinctAlleleCount).forEach(n -> action.accept(sortedAlleleCounts[2*n], sortedAlleleCounts[2*n+1]));
+    }
+
+    @FunctionalInterface
+    public interface TwoIntToDoubleFunction {
```

Comment:
Perhaps this interface should be in a utility package?.... I guess it may be used in some other context.